.kube.dev.yaml

```diff
@@ -0,0 +1,28 @@
+---
+- name: register
+  recipe: webapp
+  url: register.notprod.asl.homeoffice.gov.uk
+  image: quay.io/ukhomeofficedigital/asl-register:{{.DRONE_COMMIT_SHA}}
+  env:
+    REDIS_HOST: register-redis
+    REDIS_PASSWORD:
+      secret: true
+      name: asl-dev-redis
+      key: pass
+    SESSION_SECRET:
+      secret: true
+      name: asl-dev-session
+      key: secret
+    KEYCLOAK_REALM: Asl-dev
+    KEYCLOAK_URL: https://sso-dev.notprod.homeoffice.gov.uk/auth
+    KEYCLOAK_CLIENT: asl-dev-connect
+    KEYCLOAK_SECRET:
+      secret: true
+      name: asl-dev-auth
+      key: secret
```

Comment:
It feels like there's a lot more environment variables being used in config.js than are being defined here.